group-aws/integration.md

```diff
@@ -0,0 +1,34 @@
+# Integration Testing
+
+There are very few integration tests for EC2. Those that
+exist are in test/integration/targets
+
+## Expected test criteria
+
+* Resource creation
+* Resource creation again
+* Resource creation under check mode
+* Resource modification
+* Resource deletion
+
+When adding new features, those features should have tests.
+
+# Github PRs
+There are some PRs with tests:
```

Comment:
Added to Testing Working Group agenda https://github.com/ansible/community/issues/114#issuecomment-313017022